extensions/ql-vscode/test/data/library.qll

```diff
@@ -1,3 +1,3 @@
 predicate foo() {
- 1 == 1
-}
\ No newline at end of file
+ 1 = 1
```

Comment:
hah, how long was this broken, I wonder

Reply:
Oh...didn't even mean to commit this. I'll extract to a separate commit.